src/apps/authentication/api/utils.py

```diff
@@ -0,0 +1,25 @@
+from django.core.mail import EmailMessage
+import threading
+
+class EmailThread(threading.Thread):
+    """
+    Thread for sending an email
+    """
+
+    def __init__(self, email):
+        self.email = email
+        threading.Thread.__init__(self)
+
+    def run(self):
+        self.email.send()
+
+class Util:
```

Comment:
To abstract and global name for a class.
This class only manages the mail-theme functionality for now and it should stay like this.
Do not mix differently themed functionality into one class see SOLID principles for this. 
Considering the Above ^, rename this `Util` class into somewhat connected to mailing

Reply:
It is done